src/services/allowances/allowance-sources/rpc-allowance-source.ts

```diff
@@ -1,15 +1,16 @@
 import { ChainId, TimeString, TokenAddress } from '@types';
-import { IMulticallService } from '@services/multicall';
 import { AllowanceCheck, IAllowanceSource, OwnerAddress, SpenderAddress } from '../types';
 import { timeoutPromise } from '@shared/timeouts';
 import { filterRejectedResults } from '@shared/utils';
 import ERC20_ABI from '@shared/abis/erc20';
+import { IProviderService } from '@services/providers';
+import { Address } from 'viem';
```

Comment:
Can we do this? So that we don't confuse it with the SDK's `Address`
```suggestion
import { Address as ViemAddress } from 'viem';
```